docs/getting-started/Configuration.md

```diff
@@ -0,0 +1,72 @@
+# How to Configure Warden #
+
+Configuration options for Warden can be specified by providing a configuration
+file on the command-line.
+
+## Command-line Options ##
+
+The options below are specified on the command-line.
+
+* `--config` - A configuration file to load. For more information on the format
+  of this file, see the **Configuration Files** section. Only one configuration
+  file can be specified. Multiple uses of this argument will result in only the
+  last configuration file being read.
+
+## Configuration Files ##
+
+Configuration files are JSON formatted. They are a single JSON object
+containing configuration values.
+
+### Default Configuration File ###
+
+The configuration file, located in warden/config/defaults.json below is the default settings for Warden.
```

Comment:
This is clunky - maybe:

"This is the default configuration, which you can find in <fix this install dir>/config/defaults.json".